cmd/minishift/cmd/openshift/service.go

```diff
@@ -60,22 +46,13 @@ var serviceCmd = &cobra.Command{
 
 		service := args[0]
 
-		api := libmachine.NewClient(constants.Minipath, constants.MakeMiniPath("certs"))
-		defer api.Close()
-
-		url, err := cluster.GetServiceURL(api, namespace, service, serviceURLTemplate)
+		url, err := openshift.GetServiceURL(service, namespace, https)
```

Comment:
👍 Nice, just a call to _openshift.GetServiceURL_. Keeping just the command switch parsing here.